dali/operators/audio/nonsilence_op.h

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_AUDIO_NONSILENCE_OP_H_
+#define DALI_OPERATORS_AUDIO_NONSILENCE_OP_H_
+
+#include <cmath>
+#include <limits>
+#include <memory>
+#include <utility>
+#include <vector>
+#include "dali/core/convert.h"
+#include "dali/kernels/kernel_manager.h"
+#include "dali/kernels/signal/decibel/decibel_calculator.h"
+#include "dali/kernels/signal/moving_mean_square.h"
+#include "dali/pipeline/data/views.h"
+#include "dali/pipeline/operator/operator.h"
+
+namespace dali {
+namespace detail {
+
+const int kNumOutputs = 2;
+using OutputType = int;
+static_assert(std::is_integral<OutputType>::value,
```

Comment:
what's the point of this static assert?

Reply:
At some point there was one. It' not there anymore ;)